### ISSUE TITLE
Edit Transparency page

### ISSUE THREAD

**Issue by NathanBnm** [datenanfragen/website#238]
The following sentence should modified after #209 

>We publish the complete constitution of our association on [this website]({{< ref "verein/constitution" >}}) and on GitHub.

`website` should be replaced by `page` (already done in French translation)

**Comment by baltpeter** [comment]
Is 'this website' actually wrong here? I am using 'this' to refer to the website you are currently on and `verein/constitution` is part of *this* website, isn't it? :D

**Comment by NathanBnm** [comment]
I think that refering to "this website" implies to open a new tab with a new website. That's why I think it should be replaced by "this page".

It was all right when the association's website was separate, wasn't it?

**Comment by baltpeter** [comment]
> It was all right when the association's website was separate, wasn't it?

To be honest, I don't understand how this was different then. :D When we still had `verein.datenanfragen.de`, both the transparency and constitution page on that website, too. 

But I mean, "this page" also isn't wrong. If you think that is clearer, we can change it. \*shrug*

**Comment by NathanBnm** [comment]
> But I mean, "this page" also isn't wrong. If you think that is clearer, we can change it. *shrug*

Maybe it's a difference between languages too. I mean, in French I think it's clearer to use this. It's up to you!

**Comment by baltpeter** [comment]
After rereading the sentence again after a couple of days, I have to say I think that 'this website' is clearer in terms of establishing that we are talking about the website the user is currently on versus a different one.

Let's ask @zner0L for his opinion. If he thinks differently, he can reopen the issue.

**Comment by NathanBnm** [comment]
Well after reflexion I think you're right x)

**Comment by zner0L** [comment]
I don't really care, but if you want my opinion: I think "this page" is better than "this website". That being said, I don't think any of the options are unclear, so I'll leave this issue closed.
